runtime/drivers/druid/native_query.go

```diff
@@ -61,7 +71,9 @@ type QueryPlan struct {
 	} `json:"query"`
 }
 
-func (n *NativeQuery) Do(ctx context.Context, dr, res interface{}, queryID string) error {
+func (n *NativeQuery) Do(ctx context.Context, dr, res interface{}, queryID string, logger *zap.Logger) error {
+	logger = logger.With(zap.String("query_id", queryID))
+	logger.Debug("Executing native query", zap.Any("request", dr))
```

Comment:
It should probably only do these logs if `connector.druid.log_queries` is `true`